app/src/main/java/com/example/poste/http/RetrofitClient.java

```diff
@@ -10,7 +11,11 @@ public class RetrofitClient {
 
     public static Retrofit getRetrofitInstance() {
         if (retrofit == null) {
-            OkHttpClient client = new OkHttpClient.Builder().build();
+            HttpLoggingInterceptor logging = new HttpLoggingInterceptor();
+            logging.setLevel(HttpLoggingInterceptor.Level.HEADERS);
+            OkHttpClient client = new OkHttpClient.Builder()
+                    .addInterceptor(logging)
```

Comment:
This adds logging to our HTTP request client, for debugging purposes.

Reply:
This is a good idea.